crypto/provider_predefined.c

```diff
@@ -21,6 +22,7 @@ const struct predefined_providers_st predefined_providers[] = {
     { "fips", fips_intern_provider_init, 1 },
 #else
     { "default", ossl_default_provider_init, 1 },
+    { "base", ossl_base_provider_init, 0 },
```

Comment:
Hmmm, this means that it's only pre-defined, but not activated...  on the other hand, setting the flag to 1 only has it auto-activated if no provider has been explicitly loaded beforehand.

It's not strictly necessary to resolve that in *this* PR, and I have an idea on how to do it, if we can have the patience to wait (I don't even have a branch yet, for once)

Reply:
The default provider doesn't need this and it shouldn't be automatically loaded with the default for performance reasons.  I think it should load with the FIPS provider automatically, but that's for another PR.

Reply:
Actually, the base is in the wrong place...  will move.